services/blockchain-indexer/tests/unit/shared/dataservice/business/events.test.js

```diff
@@ -0,0 +1,167 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2023 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+/* eslint-disable import/no-dynamic-require */
+const path = require('path');
+
+const mockedBlockID = '89a9f8dd0e9d15e54268f952b2e9430e799968169376273f715480d058a67dc4';
+const mockedEvents = [{ id: 'c3a1f388859d69105241473ce9a49d831c8eb0043f717a626b2901f52ed3610c', name: 'transfer' }, { id: 'c3c120616d202a36e659248e6d384bbbf3429d4d56a6066f368aa14d16e81dfb', name: 'validatorRegistered' }];
```

Comment:
Instead of defining separate constant for the events here, may be we can directly use the event already defined under `tests/constants` directory. 
Ref: https://github.com/LiskHQ/lisk-service/blob/release/0.7.0/services/blockchain-indexer/tests/constants/events.js#L29